package-lock.json

```diff
@@ -1,12347 +1,8 @@
 {
   "name": "droidconke2022reactnative",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Im seeing lockfileversion has changed to lofckfileVersion1. Should this be a concern?

Reply:
I think it's something to do with npm versions. lockfileversion 2 is NPM 7 while 6 is lockfileversion 1. Not sure just a thought though 🤔

Reply:
> I think it's something to do with npm versions. lockfileversion 2 is NPM 7 while 6 is lockfileversion 1. Not sure just a thought though thinking

@makunomark what do you think about this? personally I think that if it won't break the project then that's okay.

Reply:
> > I think it's something to do with npm versions. lockfileversion 2 is NPM 7 while 6 is lockfileversion 1. Not sure just a thought though thinking
> 
> @makunomark @orama254 what do you think about this? personally I think that if it won't break the project then that's okay.

**UPDATE**
- For now, let's merge this PR, then create an issue to explore whether lock file versions changing will break the project (somehow)